Web/src/components/games/MentalSums/MentalSumsGame.tsx

```diff
@@ -12,74 +14,92 @@ interface MentalSumsGameProps {
 }
 
 const GameContainer = styled.div`
-  background: var(--pale-yellow);
+  background: var(--purple);
   overflow: hidden;
-  height: 100vh;
-  width: 100vw;
+  min-height: ${window.innerHeight}px;
+
   display: flex;
   justify-content: center;
   flex-direction: column;
   align-items: center;
 
+  color: white;
+  font-size: 1.2rem;
+  text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
+`;
+
+const GameplayContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  width: ${smMin};
+
+  @media (max-width: ${smMin}) {
+    width: 90vw;
+  }
+
   & > * {
-    z-index: 1;
+    margin: 0.3rem 0;
+    width: 100%;
   }
 `;
 
-const CorrectContainer = styled.div`
-  animation: ${blinkGreen} 0.5s ease-in-out 0s 1;
-  position: fixed;
-  overflow: hidden;
-  height: 100vh;
-  width: 100vw;
-  display: flex;
-  z-index: 1;
+type QuestionContainerProps = {
+  animateCorrect?: boolean;
+  animateWrong?: boolean;
+};
+
+const QuestionContainer = styled.div`
+  background: white;
+  border-radius: 15px;
+  font-size: 2rem;
+  text-align: end;
+  color: var(--dark-purple);
+
+  transition: background-color 0.5s ease-out;
+
+  & > * {
+    margin: 0.5rem 1rem;
+  }
+
+  ${({ animateCorrect }: QuestionContainerProps) => animateCorrect
```

Comment:
Why not make this just `isCorrect: boolean`, then

```tsx
${
({ isCorrect }: QuestionContainerProps) =>
  isCorrect ? css`background-color: var(--green)` : css`background-color: var(--pink)`};
}
```

Unless this is some 1984 Orwellian world where war is peace, freedom is slavery and ignorance is strength, or Schroedinger's cat question, the answer can either be correct or wrong and not both at the same time lol

Reply:
LOL I was using the original states but ya ok makes sense!!! 👍